src/components/ImageView/RegionView/LineSegmentRegionComponent.tsx

```diff
@@ -280,7 +279,7 @@ export class LineSegmentRegionComponent extends React.Component<LineSegmentRegio
             }
 
             // Construct anchors if region is selected
-            if (this.props.selected && this.props.listening && !region.locked) {
+            if (this.props.selected && this.props.listening && !region.locked && !AppStore.Instance.activeFrame?.regionSet.locked) {
```

Comment:
makes sense to create a variable `const frame = AppStore.Instance.activeFrame` above, if you're going to reuse it a few times in the function

Reply:
(or `regionSet`

Reply:
actually, don't worry. Not worth delaying this PR for such a minor thing 😄